content/docs/handling-events.md

```diff
@@ -138,17 +138,17 @@ class LoggingButton extends React.Component {
 }
 ```
 
-The problem with this syntax is that a different callback is created each time the `LoggingButton` renders. In most cases, this is fine. However, if this callback is passed as a prop to lower components, those components might do an extra re-rendering. We generally recommend binding in the constructor or using the class fields syntax, to avoid this sort of performance problem.
+O problema com esta sintaxe é que um callback diferente é criado toda vez que o `LoggingButton` é renderizado. Na maioria dos casos, tudo bem. No entanto, se esse callback for passado para componentes inferiores através das propriedades, esses componentes poderão fazer uma renderização extra. Geralmente recomendamos a vinculação no construtor ou a sintaxe dos campos de classe para evitar esse tipo de problema de desempenho.
 
-## Passing Arguments to Event Handlers {#passing-arguments-to-event-handlers}
+## Passando argumentos para manipuladores de eventos {#passing-arguments-to-event-handlers}
```

Comment:
```suggestion
Passando Argumentos para Manipuladores de Eventos
```

Reply:
@glaucia86 Should I translate the link for this title?
```suggestion
## Passando Argumentos para Manipuladores de Eventos {#passando-argumentos-para-manipuladores-de-eventos}
```

Reply:
@httpiago yes.